src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java

```diff
@@ -900,17 +900,37 @@ private AddOnCollection getPreviousVersionInfo() {
 	}
 
     
-    private List<AddOn> getUpdatedAddOns() {
+    public List<AddOn> getUpdatedAddOns() {
```

Comment:
Protected instead? Or, other extensions should be able to access/change the returned AddOn(s)?